GVFS/GVFS.Notifications/VFSForGit.Mac/StatusMenuItem/Utilities/VFSProductInfoFetcher.h

```diff
@@ -0,0 +1,16 @@
+#import <Foundation/Foundation.h>
+#import "VFSProcessRunner.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface VFSProductInfoFetcher : NSObject
+
+- (instancetype _Nullable)initWithProcessRunner:(VFSProcessRunner *)processRunner;
+- (BOOL)tryGetGitVersion:(NSString *_Nullable __autoreleasing *_Nonnull)version
+                   error:(NSError **)error;
```

Comment:
Missed a couple of _Nullable and __autoreleasing decorations for these NSError parameters

Reply:
@alameenshah are changes still needed here for this comment?